File: src/ad/Discipline.java
package ad;

public class Discipline {
	private Integer Id;
	private String Name;
	private String ShortName;
	
	public Integer getId(){
		return Id;
	}
	public void setId(Integer Id){
		this.Id=Id;
	}
	
	public String getName(){
		return Name;
	}
	public void setName(String Name){
		this.Name=Name;
	}
	
	public String getShortName(){
		return ShortName;
	}
	public void setShortName(String ShortName){
		this.ShortName=ShortName;
	}
}

File: web/js/Model/Entity/Discipline.js
"use strict";

var Discipline = function (id,name,shortName) {
    Entity.apply(this, arguments);
    this._name = name;
    this._shortName = shortName;

    this.toString = function () {
        return this._id + "\t" + this._name + "\t" + this._shortName;
    };
    this.getName = function () {
        return this._name;
    };
    this.setName = function (name) {
        this._name = name;
    };
    this.getShortName = function () {
        return this._shortName;
    };
    this.setShortName = function (shortName) {
        this._shortName = shortName;
    };
};
Discipline.prototype =  Object.create(Entity);